### ISSUE TITLE
Do not download notes multiple times

### ISSUE THREAD

**Issue by DavidFarago** [mysticcoders/voicenotes-sync#37]
My Voicenotes obsidian workflow is as follows:
1) record audio notes via Voicenotes
2) Voicenotes Sync does not remove them in Voicenotes due to Voicenotes' features like "Ask my AI"
3) Process my downloaded notes in Obsidian, which often results in moving the note some place else or removing it.

Once Voicenotes Sync is again syncing with Voicenotes, notes that have already been moved or removed in Obsidian occur again, which completely breaks my workflow. A single download of each note is the feature I seek.

Suggested technical steps: Is it possible for Voicenotes Sync to add a tag `synced_with_obsidian` to notes in Voicenotes when they are being downloaded, instead of removing the notes? And then configuring Voicenotes Sync to not download any notes from Voicenotes that already have the tag `synced_with_obsidian`?

**Comment by cvasanthk** [comment]
Yes David, I vote for this request. 
I'm not sure if 'voicenotes sync' can write back the tags to Voicenotes using the available API?

**Comment by DavidFarago** [comment]
Yes, I don't know either. I looked at the code and it seems the REST verb DELETE can be used, but I haven't seen CREATE. 

If it happens that tags cannot be created but deleted, a workaround would be to somehow have Voicenotes always create the tag `to-be-synced-with-obsidian`, and Voicenotes Sync downloads only those and removes that tag.

**Comment by kinabalu** [comment]
You should be able to use the "Exclude Tags" feature to achieve this which would hopefully mean they wouldn't sync
